packages/lesswrong/themes/userThemes/darkMode.ts

```diff
@@ -270,6 +270,7 @@ export const darkModeTheme: UserThemeSpecification = {
       votedCandidate: "#8A5751",
       ctaBackground: "#64342E",
       ctaText: "#FFF",
+      rhsLink: '#d54646',
```

Comment:
@agnesstenlund I thought the standard red was too dark for the RHS links in dark mode, so I picked a lighter red. It doesn't really look right to me though, so feel free to change this color and pick a better one :)